src/Lifi.ts

```diff
@@ -28,18 +29,31 @@ import {
 
 class LIFI {
   private activeRoutes: ActiveRouteDictionary = {}
-
   private config: Config = getDefaultConfig()
 
-  getConfig = () => {
+  /**
+   * Get the current configuration of the SDK
+   * @return {Config} - The config object
+   */
+  getConfig = (): Config => {
     return this.config
   }
 
-  setConfig = (configUpdate: ConfigUpdate) => {
+  /**
+   * Set a new confuration for the SDK
+   * @param {ConfigUpdate} configUpdate - The new configuration. Same as interface Config but with optional values.
```

Comment:
Maybe make it shorter and simpler? 
```suggestion
   * @param {ConfigUpdate} configUpdate - The configuration fields that should be updated.
```
If not then it should at least be "real" sentences.